src/docs/product/accounts/quotas/manage-replay-quota.mdx

```diff
@@ -0,0 +1,86 @@
+---
+title: Manage Your Replay Quota
+keywords: ["best practices"]
+sidebar_order: 40
+description: “Learn how to use Sentry’s tools to manage the amount and types of replays you pay for.”
+---
+
+Because sending all your replays to Sentry may rapidly consume your quota, we provide tools to help you manage the *amount and types* of replays that are accepted.
+
+Knowing how to apply the proper filters, SDK configuration, and rate limits is an iterative and on-going process, but these tips will show you how to get the most out of your error events. They’re ordered from easiest or least time-consuming, to most challenging or potentially time-consuming. The first three are all things you can do in [sentry.io](https://sentry.io/), whereas the remaining one is something you'll do in the SDK.
+
+## Before You Begin: Check Your Quota Usage {#3-replay-usage-stats}
+
+You can look at your replays in aggregate in the "Usage Stats" tab of the **Stats** page to understand the breakdown of your incoming replays and see which projects are consuming your [quota](The monthly number of events that you pay Sentry to track.). This may help you figure out where you need to further fine-tune your SDK filters and configuration.
+
```

Comment:
```suggestion
You can look at your replays in aggregate in the "Usage Stats" tab of the **Stats** page to understand the breakdown of your incoming replays and see which projects are consuming your quota. This may help you figure out where you need to further fine-tune your SDK filters and configuration.

```